dashboard/modules/reporter/tests/test_reporter.py

```diff
@@ -244,7 +243,7 @@ def test_worker_stats():
     wait_for_condition(test_worker_stats, retry_interval_ms=1000)
 
 
-def test_report_stats():
+def test_report_stats(shutdown_only):
```

Comment:
you don't need it here right? no ray cluster is started.

Reply:
You are correct since we used `dashboard_agent = MagicMock()` to init the dashboard agent and we don't test dashboard head in the test case